bundles/binding/org.openhab.binding.mios/src/main/java/org/openhab/binding/mios/internal/config/ServiceAliases.properties

```diff
@@ -109,3 +109,4 @@ Tuning1=urn:micasaverde-com:serviceId:Tuning1
 VideoAdjustment1=urn:micasaverde-com:serviceId:VideoAdjustment1
 Volume1=urn:micasaverde-com:serviceId:Volume1
 WMC1=urn:micasaverde-com:serviceId:WMC1
+RGBController1=urn:upnp-org:serviceId:RGBController1
```

Comment:
Agreed - the plugin developer shouldn't have used that. However, it's the most popular RGB plugin for MIOS, and the one recommended for the new Aeotec bulb. https://apps.mios.com/plugin.php?id=6686